### PR TITLE
Update raw values for scorecard summary factors

### PR DESCRIPTION
**Pull request by justinebreuch**
## Description

Addresses: Finalize scorecard content + styling

Updates some styling on the scorecard page to closer align with other pages. 

Contextualizes numbers in scorecard factors for now. These numbers will be run through UXR testing to determine whether we should show them at all. 

### New
Formats the demographic numbers to text rather than raw numbers. See screenshot. 

## Testing and Reviewing
Screen Shot 2022-08-22 at 4 04 15 PM

### CODE REVIEW

**Comment by kailamjeter** [comment]
I know content isn't finalized but for now can you change that first sentence to:

'In addition to information from your water system, your score is also based on these statistics for your zip code:"

Just to explicitly mention that these numbers are for zip.

File: client/src/components/ScorecardSummaryPanel.vue [hunk 1]
```diff
@@ -63,13 +73,47 @@ export default defineComponent({
   },
   computed: {
     homeAgeComparison(): string | null {
-      return this.formatNumber(this.demographicDataState?.data?.averageHomeAge);
+      const homeAge = this.roundNumber(this.demographicDataState?.data?.averageHomeAge);
```
Comment (kailamjeter) [inline client/src/components/ScorecardSummaryPanel.vue:76]: nit can we check if this is null before rounding and just return?

same below for income

**Review by kailamjeter — APPROVED** [review]

File: client/src/components/ScorecardSummaryPanel.vue (hunk 1 above)
Comment (justinebreuch) [reply client/src/components/ScorecardSummaryPanel.vue:76]: Round number does that - it checks if it's null and just returns null in that case

**Comment by justinebreuch** [comment]
> I know content isn't finalized but for now can you change that first sentence to:
> 
> 'In addition to information from your water system, your score is also based on these statistics for your zip code:"
> 
> Just to explicitly mention that these numbers are for zip.

Good idea. Updated

**Pull request merged**